lib/ansible/modules/group.py

```diff
@@ -651,6 +652,9 @@ def group_add(self, **kwargs):
         return self.execute_command(cmd)
 
     def group_del(self):
+        if self.module.params['force']:
+            self.module.fail_json(msg='force is not a valid option for this platform.')
```

Comment:
`force` alone is not enough, you should specify that you cannot force delete a group